eventarc/generic/Dockerfile

```diff
@@ -16,7 +16,7 @@
 
 # Use the official lightweight Ruby image.
 # https://hub.docker.com/_/ruby
-FROM ruby:2.7-slim
+FROM ruby:2.5-slim
```

Comment:
I don't think we want to downgrade to Ruby 2.5. Especially as this is the image we're deploying to the Cloud Run service, not the Ruby that's running the tests. For a customer-visible sample, better to show a current Ruby.

Same with the pubsub image below.